packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SubjectGroupViewer/SubjectGroupViewer.js

```diff
@@ -21,6 +27,29 @@ const Container = styled.div`
   }
 `
 
+/*
+Note on Subject Viewer sizing/fitting:
+- The grid should fit the height OR width of the available visible space.
+- This is implemented with the container div AND the <svg> having
+  width/height=100% (which 'fits') an optional max-width/height (which
+  'restricts').
+- Curious note: the max-width/height has to be repeated in <svg> due to
+  Safari 12. If we ignore Safari, we only need max-width/height on the
+  container div, not the <svg>
+ */
+const SVG = styled.svg`
+  width: 100%;
+  height: 100%;
+
+  ${props => props.gridMaxWidth
+    ? css`max-width: ${props.gridMaxWidth};`
+    : ''}
+  ${props => props.gridMaxHeight
+    ? css`max-height: ${props.gridMaxHeight};`
+    : ''}
+`
+SVG.displayName = 'styledSvg'  // Required for testing
```

Comment:
Styled components generates unique names for the components wrapped by styled. It can be disabled by way of the babel plugin: https://styled-components.com/docs/tooling#control-the-components-displayname. I'm not sure if or what impact it might have to overwrite the display name manually like this. If this is just for testing purposes, however, you can export the component itself and import it so you can find the component instance itself rather than find by display name.